### DISCUSSION TITLE
How to use more than one instance of ketcher on the same page at once.

### DISCUSSION THREAD

**Discussion by OliverDudgeon** [epam/ketcher#2313] [Q&A]
I have a react app and want to use multiple instances of the ketcher on the same page at once. Thos doesn't seem possible since both would need to use the same `window.ketcher` global. **Is there anyway to configure the global used by a ketcher instance?**

**Comment by yulei-chen** [comment]
Hi, I think Ketcher now supports multiple instances, but some features will disappear, such as context menu, saving as image, etc. We have an issue for that - #2250.
If you want to visit every instance by global `window`, you could try this way

```js
<Editor
  // ...
  onInit={(ketcher) => {
      window.ketcher1 = ketcher
      // ...
  }}
/>
```

```js
<Editor
  // ...
  onInit={(ketcher) => {
      window.ketcher2 = ketcher
      // ...
  }}
/>
```
